### PR TITLE
quick fix for deploy UI with v0.5

### PR DESCRIPTION
**Pull request by kunmingg**
* Skip password input check when choose "Setup Endpoint later"
* rename error from ```spec not found```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2939)

### CODE REVIEW

**Comment by lluunn** [comment]
which part is "skip endpoint"?

**Comment by kunmingg** [comment]
@lluunn 
One option on UI called ```Setup Endpoint later```:
User won't have an working ingress, and they don't need to create oauth or input username/password

File: components/gcp-click-to-deploy/src/DeployForm.tsx [hunk 1]
```diff
@@ -490,7 +490,8 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
           return;
         }
       }
-    } else {
+    }
```
Comment (lluunn) [inline components/gcp-click-to-deploy/src/DeployForm.tsx]: So skip endpoint will not IngressType.Iap nor IngressType.BasicAuth?
Can we do else if (this.state.ingress === IngressType.BasicAuth) here, and then 
else {
// skip endpoint
log something
}

Comment (kunmingg) [reply components/gcp-click-to-deploy/src/DeployForm.tsx]: Log here would be user facing as this part runs at user browser.
Do you mean print some messages for  "Setup Endpoint later" case?
There's a message field for each mode, we can use that to explain behavior of user's choice

**Pull request merged**